tasks/check_ssh.yml

```diff
@@ -1,11 +0,0 @@
-- name: Waiting for SSH to be up
-  wait_for:
-    host: "{{ item.public_dns_name }}"
-    port: 22
-    delay: 5
-    timeout: 300
-    state: started
-  with_items: "{{ ec2.instances }}"
-
-- pause:
-    seconds: 15
```

Comment:
Curious why the removal of check ssh?

Reply:
@oblodgett It looks like it's moved to another place in the code, if I'm not mistaken?

https://github.com/alliance-genome/agr_ansible_developers/pull/13/files#diff-a385ec69082237ced4df664197bcaeb091c951cd67924eb59777cb19a9cda7d6R99

Reply:
@christabone @oblodgett Yes, rest assured that the ssh check is still there (excactly where @christabone linked to). I simply removed the subtask file and put the content (the ssh check task) straight in the playbook, as it is not being used anywhere else but that playbook (and only once) and was thus having it in a separate task file was only making the flow harder to follow.